packages/cozy-sharing/locales/en.json

```diff
@@ -48,6 +48,7 @@
       "confirm": "Confirm",
       "reject": "Reject",
       "cancel": "Cancel",
```

Comment:
do reject and confirm still needed?

Reply:
yes, they are still need for the dialogs buttons

https://github.com/cozy/cozy-libs/pull/1353/files#diff-51e44a6c84e55e3a0b74af33011956c42b4c915f06f59a2fc39b6a928d6c167eL122-L133